metal-k8s.yml

```diff
@@ -48,13 +49,20 @@
       tags: ['kube-prometheus']
     - role: kube_prometheus
       tags: ['kube-prometheus']
+    - role: kube_prometheus_node_exporter
+      tags: ['kube-prometheus', 'node-exporter']
     - role: kube_heapster
       tags: ['heapster']
     - role: kube_elasticsearch
       tags: ['elasticsearch']
     - role: kube_metrics_server
       tags: ['metrics-server']
 
+- hosts: kube-node
```

Comment:
Shouldn't this go on `kube-node`, `kube-master` (so actually `k8s-cluster`) *and* `etcd`?